react/src/base/inputs/DatePicker.stories.js

```diff
@@ -33,30 +36,49 @@ props table below for available customization options.
 
 export const defaultStory = () => (
   <SprkInputContainer>
-    <SprkDatePickerInput
-      name="date"
-      placeholder="01/01/2019"
-      errorMessage="Incorrect date."
-      label="Date"
-    />
+    <div className="sprk-b-InputContainer__icon-container">
+      <SprkLabel for="datepicker-1">Date</SprkLabel>
```

Comment:
hey @guern1kn , react-dom.development.js:88 Warning: Invalid DOM property `for`. Did you mean `htmlFor`

Reply:
YUP! Sure did.... Dang it, React.